server/leafnode.go

```diff
@@ -252,6 +256,18 @@ func validateLeafNode(o *Options) error {
 	if o.LeafNode.Port == 0 {
 		return nil
 	}
+
+	// If MinVersion is defined, check that it is valid.
+	if mv := o.LeafNode.MinVersion; mv != _EMPTY_ {
+		if _, _, _, err := versionComponents(mv); err != nil {
```

Comment:
check version components be at or above v2.8.0?
More for our users than for us

Reply:
Sorry, but I don't understand this comment. Are you saying that this should be the error message logged and returned to the remove?

Reply:
I am wondering if we should check the configured version to be of a value that is actually usable.
Meaning 2.7.0 should not be a value one can configure.

Reply:
I'd add that check for our users benefit.